spec/lib/govuk_healthcheck/active_record_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "spec_helper"
+require "govuk_app_config/govuk_healthcheck"
+require_relative "shared_interface"
+
+RSpec.describe GovukHealthcheck::ActiveRecord do
+  module ActiveRecord
```

Comment:
Do we need this if we have the `allow(...)` on `ActiveRecord::Base` below?

Reply:
This has been changed to a better approach now.

Reply:
(but the answer is "yes" because you can't use `allow` to stub an unimplemented method)